### PR TITLE
Fix geo_handling and cm_transfer to take sessions rather than args

### PR DESCRIPTION
**Pull request by bhazelton**
Fixes many tests that break otherwise if the default_db is not in testing mode.

Also fixed a few bugs to make write_antenna_location_file.py work.

### CODE REVIEW

File: Deprecated/stop_all_C7.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from __future__ import absolute_import, division, print_function
 
-from hera_mc import mc, cm_utils, part_connect, cm_handling, geo_handling, cm_hookup, cm_part_revisions
+from hera_mc import mc, cm_utils, part_connect, cm_handling, cm_hookup, cm_part_revisions
```
Comment (pkgw) [inline Deprecated/stop_all_C7.py]: Might make sense not to touch the scripts in `Deprecated/` once they've served their purpose? Obviously not a big deal though.

File: hera_mc/cm_transfer.py [hunk 2]
```diff
@@ -15,37 +15,40 @@
 import csv
 
 
-def package_db_to_csv(args):
+def package_db_to_csv(session=None, tables='all', base=False, maindb=False):
     """This will get the configuration management tables from the database
        and package them to csv files to be read by initialize_db_from_csv"""
```
Comment (pkgw) [inline hera_mc/cm_transfer.py]: Docstring descriptions of the new arguments would be useful here.

File: hera_mc/cm_transfer.py [hunk 3]
```diff
@@ -54,14 +57,14 @@ def package_db_to_csv(args):
             os.remove(tmp_filename)
 
 
-def initialize_db_from_csv(args):
-    """This entry module provides a double-check entry point to read the csv files and 
+def initialize_db_from_csv(session=None, tables='all', base=False, maindb=False):
+    """This entry module provides a double-check entry point to read the csv files and
        repopulate the configuration management database.  It destroys all current entries,
        hence the double-check"""
```
Comment (pkgw) [inline hera_mc/cm_transfer.py]: Docstrings again.

File: hera_mc/cm_transfer.py [hunk 4]
```diff
@@ -136,8 +140,8 @@ def __initialization(args):
             print('Initialization for %s not found' % (table))
             use_table.remove(table)
         else:
-            if args.maindb:
-                if dbkey != args.maindb:
+            if maindb:
+                if dbkey != maindb:
```
Comment (pkgw) [inline hera_mc/cm_transfer.py]: Oh, seeing what this `maindb` variable is ... I do not think it has a very good name. But that's an issue for another PR.

File: Deprecated/stop_all_C7.py (hunk 1 above)
Comment (bhazelton) [inline Deprecated/stop_all_C7.py]: Yeah, I fixed this when it was not in deprecated, but when I rebased on master it had been moved. I can remove it if really desired, but it's importing something that's not used, so it doesn't really change anything.

File: hera_mc/cm_transfer.py (hunk 2 above)
Comment (bhazelton) [inline hera_mc/cm_transfer.py]: yes, I'll add them.

File: hera_mc/cm_transfer.py (hunk 4 above)
Comment (bhazelton) [inline hera_mc/cm_transfer.py]: yeah, it took me a while to figure it out and it's another one of those times where it can have different types, either boolean or a string which is weird. but I agree, it should be changed in a different PR.

Comment (david-deboer) [inline hera_mc/cm_transfer.py]: Right - this is already an issue for the fix maindb thing.
